### PR TITLE
Car page api.

### PR DESCRIPTION
**Pull request by SerezhaGRig**

### CODE REVIEW

File: handler.js [hunk 1]
```diff
@@ -1,41 +1,191 @@
...
+            UpdateExpression=UpdateExpression + `${key} = ${map[key]}, `
+            ExpressionAttributeValues[map[key]]=updVals[key]
+        }
+    }
+    UpdateExpression=UpdateExpression.substring(0,UpdateExpression.length-2)
+    const params = {
+        TableName: CAR_TABLE,
+        Key: {
+            login,
+            car_id:id
```
Comment (Karlen92) [inline handler.js:64]: use Camel case

File: handler.js [hunk 2]
```diff
@@ -1,41 +1,191 @@
...
-const querystring = require('querystring');
+const  AWS = require("aws-sdk");
+const uuid = require('uuid');
+AWS.config.setPromisesDependency(require('bluebird'));
+const docClient = new AWS.DynamoDB.DocumentClient();
+
+
+const CAR_TABLE = "Cars"
+const MODELS_TABLE = "Models"
+const BRANDS_TABLE = "Brands"
```
Comment (Karlen92) [inline handler.js:10]: use from environment

File: handler.js [hunk 3]
```diff
@@ -1,41 +1,191 @@
...
-        2
-    ),
-  };
+        UpdateExpression,
+        ExpressionAttributeValues,
+        ReturnValues:"UPDATED_NEW"
+    };
+    try {
+        await docClient.update(params).promise()
+        response =  {
```
Comment (Karlen92) [inline handler.js:72]: just return here

File: handler.js [hunk 4]
```diff
@@ -1,41 +1,191 @@
...
+            statusCode: 200,
+            body: "compleated!",
+        };
+    }
+    catch (err) {
+        response =  {
```
Comment (Karlen92) [inline handler.js:78]: just return

File: handler.js [hunk 5]
```diff
@@ -1,41 +1,191 @@
...
+        TableName: CAR_TABLE,
+        Key: {
+            login,
+            car_id:id
+
+        }
+    };
+    try {
+        let data = await docClient.get(params).promise()
+        response =  {
```
Comment (Karlen92) [inline handler.js:100]: you use this object too often, maybe you could create a helper function that will return this object?

File: handler.js [hunk 6]
```diff
@@ -1,41 +1,191 @@
 "use strict";
```
Comment (Karlen92) [inline handler.js:1]: Don't add all your code in a single file, divide by logical components

File: serverless.yml [hunk 7]
```diff
@@ -76,6 +96,129 @@ resources:
...
+        - ApiGatewayRestApi
+      Type: AWS::ApiGateway::Authorizer
+      Properties:
+        Name: cognito-authorizer
+        IdentitySource: method.request.header.Authorization
+        RestApiId:
+          Ref: ApiGatewayRestApi
+        Type: COGNITO_USER_POOLS
+        ProviderARNs:
+          - Fn::GetAtt: [ CognitoUserPool, Arn ]
```
Comment (Karlen92) [inline serverless.yml:147]: use GetAtt in a different form

**Review by Karlen92 — CHANGES_REQUESTED** [review]

**Comment by SerezhaGRig** [comment]
=-o=

**Pull request closed without merging**